cmd/agent/common/path/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/DataDog/datadog-agent/cmd/agent/common/path
+
+go 1.20
+
+replace github.com/DataDog/datadog-agent/pkg/util/executable => ../../../../pkg/util/executable
+
+replace github.com/DataDog/datadog-agent/pkg/util/log => ../../../../pkg/util/log
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏can you merge those a single `replace` instruction ?

Reply:
Sure